airflow/dags/rt_loader_files/calitp_files_process.py

```diff
@@ -46,8 +46,8 @@ def main(execution_date, **kwargs):
 
     ser_id = raw_res["name"].str.split("/")
 
-    raw_res["calitp_itp_id"] = int(ser_id.str.get(-3))
-    raw_res["calitp_url_number"] = int(ser_id.str.get(-2))
+    raw_res["calitp_itp_id"] = ser_id.str.get(-3).astype(int)
+    raw_res["calitp_url_number"] = ser_id.str.get(-2).astype(int)
```

Comment:
Just curious what led to making this change.

Reply:
Initially I thought that it was a schema issue, since the itp ids w/o this change are AFAICT strings. Now that I know it was needing to drop the header, I'm not as sure, but I included when testing just to have things bolted down / matching the schema